_includes/footer.html

```diff
@@ -1,12 +1,63 @@
-<footer class="usa-footer usa-footer-medium usa-section background-gray-dark" role="contentinfo">
+<footer class="usa-footer usa-section background-medium" role="contentinfo">
   <div class="usa-grid">
-    <h2>Drop us a line.</h2>
-    <p>To share your thoughts, email us anytime at <a href="mailto:{{ site.footer.email }}">{{ site.footer.email }}</a>.</p>
+    <div class="footer-logo-links">
+    {% include icons/oes.svg %}
```

Comment:
@wslack  yes, I tried to replace it with the .png file but it did not react well to that